### PR TITLE
Af feature selfguidedschema

### PR DESCRIPTION
**Pull request by alefaissal**
# SelfGuided Tour Schema

- Add image to the schema

### CODE REVIEW

File: client/js/components/AudioPlayer/AudioPlayer.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  setVolume(volume) {
+    RNSoundPlayer.setVolume(volume);
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.secondContainer}>
+          <Button title="Play" onPress={() => this.onPlay()} color="red" />
```
Comment (TienSFU25) [inline client/js/components/AudioPlayer/AudioPlayer.js]: onPress=onPlay?

earlier in the file you can have async onPlay => {...}

File: client/js/components/AudioPlayer/AudioPlayer.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  loadingUrl() {
+    RNSoundPlayer.loadUrl(url);
+  }
+
+  componentWillUnmount() {
+    _onFinishedLoadingSubscription.remove();
+    _onFinishedLoadingURLSubscription.remove();
+  }
+
+  play() {
```
Comment (TienSFU25) [inline client/js/components/AudioPlayer/AudioPlayer.js]: where are these play/pause/resume functions used?

File: client/js/components/AudioPlayer/AudioPlayer.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import {StyleSheet, View, Button} from 'react-native';
+import Player from 'react-native-streaming-audio-player';
+import SoundPlayer from 'react-native-sound-player';
+import {NativeModules, NativeEventEmitter, Platform} from 'react-native';
+const {RNSoundPlayer} = NativeModules;
+
+export default class AudioPlayer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {currentTime: 0};
```
Comment (TienSFU25) [inline client/js/components/AudioPlayer/AudioPlayer.js]: whats the currentTime in the state doing?

File: client/js/components/AudioPlayer/AudioPlayer.js (hunk 1 above)
Comment (alefaissal) [reply client/js/components/AudioPlayer/AudioPlayer.js]: Removed from this branch

File: client/js/components/AudioPlayer/AudioPlayer.js (hunk 2 above)
Comment (alefaissal) [reply client/js/components/AudioPlayer/AudioPlayer.js]: Removed from this branch

File: client/js/components/AudioPlayer/AudioPlayer.js (hunk 3 above)
Comment (alefaissal) [reply client/js/components/AudioPlayer/AudioPlayer.js]: Removed from this branch

**Pull request merged**